docs/visualize/timelion-deprecation.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[timelion-deprecation]]
+== Timelion App Deprecation
+
+*With 8.0*, Timelion app is going to be deprecated so there are some actions you should take to copy your existing Timelion worksheets to a dashboard.
+This guide is designed to help you through this process. 
+
+One important note here is that it is going to affect *only the Timelion application*. 
+
+[role="screenshot"]
+image::images/timelion-app.png[]
```

Comment:
This looks like an older version of the Kibana UI? I would remove all of the navigation, so that it looks like:

![timelion_ui](https://user-images.githubusercontent.com/38990715/89574988-a8118780-d7f2-11ea-98b4-2ba7aa506876.png)